src/server/database/setupDB.js

```diff
@@ -21,28 +21,30 @@ async function reset({db, isUpdate}) {
     console.log(`>>Creating Database: ${db}`);
     await r.dbCreate(db);
   }
-  let tables = await r.db(db).tableList();
+  const tables = await r.db(db).tableList();
   if (!isUpdate) {
     console.log(`>>Dropping tables on: ${db}`);
     await Promise.all(tables.map(table => r.db(db).tableDrop(table)));
   }
-  tables = await r.db(db).tableList();
   console.log(`>>Creating tables on: ${db}`);
   await Promise.all(database.map(table => {
-    if (tables.indexOf(table.name) === -1) {
-      return r.db(db).tableCreate(table.name)
+    if (!isUpdate || tables.indexOf(table.name) === -1) {
```

Comment:
We're only creating the table if we're doing a reset or the table doesn't exist. I'm just avoiding the second call to `tableList` if we're doing a reset: we know there's nothing there.